tool/flutter_ci_script_dev.sh

```diff
@@ -6,30 +6,30 @@ DIR="${BASH_SOURCE%/*}"
 source "$DIR/flutter_ci_script_shared.sh"
 
 declare -ar PROJECT_NAMES=(
-    "add_to_app/fullscreen/flutter_module"
-    "add_to_app/prebuilt_module/flutter_module"
-    "add_to_app/plugin/flutter_module_using_plugin"
     "add_to_app/books/flutter_module_books"
+    "add_to_app/fullscreen/flutter_module"
     "add_to_app/multiple_flutters/multiple_flutters_module"
-    "animations"
+    "add_to_app/plugin/flutter_module_using_plugin"
+    "add_to_app/prebuilt_module/flutter_module"
+    # "animations"
+    "experimental/desktop_photo_search"
+    "experimental/federated_plugin/federated_plugin"
+    "experimental/web_dashboard"
     "flutter_maps_firestore"
     "infinite_list"
     "ios_app_clip"
-    "isolate_example"
+    # "isolate_example"
```

Comment:
Can we add a comment saying why we're (temporarily) skipping this? Otherwise, people will be afraid to uncomment it AND remove it completely, because they won't know why it's skipped.

Reply:
(Alternately, just remove the line altogether.)

Reply:
Good point. I'll add comments -- we've done that before, and it's how I knew to add `multiple_flutters_module` back to the list.